services/movie.py

```diff
@@ -0,0 +1,46 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None

```

Comment:
If argument can be None, you should show it in type hint
```suggestion
        genres_ids: list[int] | None = None,
        actors_ids: list[int] | None = None

```